flink-ml-lib/src/test/java/org/apache/flink/ml/feature/HashingTFTest.java

```diff
@@ -83,15 +81,9 @@ public class HashingTFTest extends AbstractTestBase {
 
     @Before
     public void before() {
-        Configuration config = new Configuration();
-        config.set(ExecutionCheckpointingOptions.ENABLE_CHECKPOINTS_AFTER_TASKS_FINISH, true);
-        env = StreamExecutionEnvironment.getExecutionEnvironment(config);
-        env.getConfig().enableObjectReuse();
-        env.setParallelism(4);
-        env.enableCheckpointing(100);
-        env.setRestartStrategy(RestartStrategies.noRestart());
+        env = TestUtils.getExecutionEnvironment();
         tEnv = StreamTableEnvironment.create(env);
-        DataStream<Row> dataStream = env.fromCollection(INPUT);
+        DataStream<Row> dataStream = env.fromCollection(INPUT, Types.ROW(Types.LIST(Types.STRING)));
```

Comment:
So users need to specify the type info by themselves, right? if no type info is specified, then kryo is used.

Reply:
Yes that is correct.